package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "payment-sdk",
+  "author": "",
+  "license": "ISC",
+  "version": "1.0.0",
+  "description": "",
+  "homepage": "https://github.com/XcooBee/payment-sdk-js#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/XcooBee/payment-sdk-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/XcooBee/payment-sdk-js/issues"
+  },
+  "main": "dist/node/node.js",
+  "module": "dist/node/node.js",
+  "browser": "dist/browser/browser.js",
+  "scripts": {
```

Comment:
It would be good to make sure the TS Types are available to the developer. I have not done this before. I think https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html documents how to do this but I haven't read it yet.